gossip/discovery/discovery_test.go

```diff
@@ -1684,6 +1704,77 @@ func TestPeerIsolation(t *testing.T) {
 	assertMembership(t, instances, peersNum-1)
 }
 
+func TestMembershipAfterExpiration(t *testing.T) {
+	// Scenario:
+	// Start 3 peers (peer0, peer1, peer2). Set peer0 as the anchor peer.
+	// Stop peer0 and peer1 for a while, start them again and test if peer3 still gets full membership
```

Comment:
who is peer3?

Reply:
@yacovm thank you for catching it. Forgot to change peer3 to peer2 after reducing from 4 peers to 3 peers. Same for other comments about peer3.